packages/localizations/src/es-ES.ts

```diff
@@ -698,8 +698,8 @@ export const esES: LocalizationResource = {
         title: 'Cuentas conectadas',
       },
       dangerSection: {
-        deleteAccountButton: 'Delete Account',
-        title: 'Account termination',
+        title: 'Peligro',
```

Comment:
Question: Why is this translated as `Danger` instead of just `Delete Account` as it is in the `en-US` file?
```suggestion
        title: 'Eliminar cuenta',
```